README.md

```diff
@@ -40,6 +39,17 @@ jobs:
           commit-message: ''
 ```
 <!-- end usage -->
+
+## Note: Token
+An authentication token is required in order to fetch the repository and add reactions on the comment.
+If you use the action in a personal repository, you can create a [PAT](https://github.com/settings/tokens) (Personal access token).
+If you use the aciton in an organisation, you probably want to use a PAT from an organizational user.
```

Comment:
```suggestion
If you use the action in an organisation, you probably want to use a PAT from an organizational user.
```

Reply:
organisation/organization is of course a matter of choice, but I think as a company we use american spelling (and you also write 'organizational'